gluten-core/src/main/scala/org/apache/gluten/GlutenPlugin.scala

```diff
@@ -267,6 +268,7 @@ private[gluten] class GlutenExecutorPlugin extends ExecutorPlugin {
 
   /** Initialize the executor plugin. */
   override def init(ctx: PluginContext, extraConf: util.Map[String, String]): Unit = {
+    CodedInputStreamClassInitializer
```

Comment:
Can we give an API for this? E.g.,

```scala
CodedInputStreamClassInitializer.modifyDefaultRecursionLimitUnsafe(100000)
```

Reply:
@zhztheplayer, I feel it may be not necessary to expose the setting API since the value we set is large enough. I prefer hiding the detail in this piece of code. Thanks!

Reply:
Thank you for the explanation. It's OK if we don't pass a number. Though I would suggest having an method for this purpose than relying on class initialization, e.g.,

```
CodedInputStreamClassInitializer.modifyDefaultRecursionLimitUnsafe()
```

Code like `CodedInputStreamClassInitializer` can be less intuitive and perhaps could cause IDEs to complain about the writing or so.